libcaf_test/caf/test/runner.cpp

```diff
@@ -68,7 +67,15 @@ int runner::run(int argc, char** argv) {
     return EXIT_SUCCESS;
   }
   default_reporter->start();
+  auto enabled = [](const std::regex& selected,
+                    const std::string_view& search_string) {
```

Comment:
It's idiomatic to pass `string_view` as value, since it's just a pointer and a size:

```suggestion
                    std::string_view search_string) {
```